ccm_web/client/src/components/TableCaption.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core'
+
+const useStyles = makeStyles(() => ({
+  srOnly: {
+    position: 'absolute',
+    width: '1px',
+    height: '1px',
+    padding: '0',
+    margin: '-1px',
+    overflow: 'hidden',
+    clip: 'rect(0,0,0,0)',
+    border: '0'
+  }
+}))
```

Comment:
I pulled this from `SectionSelectorWidget`. Maybe we can unify these, but the types of elements they are applied to are different (`caption` versus `span`).